index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css" type="text/css"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> 
+    <title>Alef's Portfolio</title>
+</head>
+<body>
+
+    <header>
+        <a href="#">
+            <img src="./assets/svg/Logo-Placeholder.svg" alt="site logo">
+        </a>
+        <nav> 
+            <a href="#" class="header--menu--icon">
+                <img src="./assets/svg/Icon-Menu.svg" alt="menu icon">
+            </a>
+        </nav>    
+    </header>
+
+    <main>
+        <section class="headline--container">
+            <div class="headline--title">
+                <h1>Aléf Garrido</h1>
+                <h2>Software Developer</h2>
+                <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+            </div>
+            <span class="devider">
+                <img src="./assets/svg/Devider.svg" alt="Section devider">
+            </span>
+            <section class="skills--container">
+                <ul class="skills--item bullet--icons">
```

Comment:
- [ ] OPTIONAL: It could be more semantically convenient to make the skills--item class elements li-tag elements and wrap them all with a ul-tag.